bank-sdk/example-app/src/main/java/net/gini/android/bank/sdk/exampleapp/core/ExampleUtil.kt

```diff
@@ -6,6 +6,8 @@ import net.gini.android.capture.network.model.GiniCaptureSpecificExtraction
 import net.gini.android.core.api.models.SpecificExtraction
 
 object ExampleUtil {
+    const val DOCUMENT = "document"
```

Comment:
[Suggestion] I prefer to prefix keys for input extras with `EXTRA_IN_...` to distinguish them from keys for output extras.

[Suggestion] I also prefer to scope keys for extras to their activity even if the same type of extra is used by different activities. So in our case I would suggest to add an `EXTRA_IN_OPEN_WITH_DOCUMENT` to `MainActivity` and another one to `CaptureFlowHostActivity`. I know it's duplication, but I prefer this style because by listing all extra keys in the activity's file it is more obvious which input/output extras an activity supports.

Reply:
Thank you for the suggestion - I applied it, but called the one in `MainActivity` differently since that one was required to be public, and was conflicting with another one.

Reply:
Oh, right global constant names have to be unique. Then what about moving the extras to the respective companion objects of `MainActivity` and `CaptureFlowHostActivity`? Then uniqueness is provided by the class name + constant name, e.g.:
```
class MainActivity {
    companion object {
        const val EXTRA_IN_OPEN_WITH_DOCUMENT = "EXTRA_IN_OPEN_WITH_DOCUMENT"
    }
}
```
This would also allow code completion to find the extra when we write `MainActivity.`:
![Screenshot 2024-03-27 at 18 08 20](https://github.com/gini/gini-mobile-android/assets/13149498/972b9ac8-4857-4b13-8362-dbdaf6859e1d)

Reply:
Done, I've moved both constants in `CaptureFlowHostActivity`, for consistency's sake.